### ISSUE TITLE
ESLint errors for inputs and outputs in new n8n versions

### ISSUE THREAD

**Issue by mahdizargarzadeh** [ivov/eslint-plugin-n8n-nodes-base#199]
Hi n8n team,

I’m experiencing ESLint errors when defining the inputs and outputs properties in custom nodes for the latest n8n versions.

Previously, the standard format was:

```
inputs: ['main'],
outputs: ['main'],
```

However, in the newer n8n versions, the recommended format appears to be:

```
inputs: [NodeConnectionType.Main],
outputs: [NodeConnectionType.Main],
```

The issue is that:

Using the old format (['main']) now triggers ESLint errors.

Using the new format ([NodeConnectionType.Main]) also triggers ESLint errors with the current eslint-plugin-n8n-nodes-base rules.

These errors seem to originate from the following files in the plugin:

`node-class-description-inputs-wrong-regular-node`

`node-class-description-outputs-wrong`

This makes it difficult to satisfy both the linting rules and the requirements of the latest n8n versions.

It would be great if the ESLint rules could be updated or guidance provided to ensure custom nodes are compatible with the new inputs and outputs format without causing lint errors.

For reference, a similar issue was reported in 2024: [#194](https://github.com/ivov/eslint-plugin-n8n-nodes-base/issues/194).

Thank you for your support!
